importer/tools/export.hs

```diff
@@ -169,6 +169,19 @@ processOneThingToTarEntries repo thing = do
     -- Get all of the files associated with the group, as entries
     runConduit $ groupIdToFiles groupId .| CL.mapMaybeM (checkoutObjectToTarEntry repo) .| CL.consume
 
+-- | Check a list of strings to see if any of them are files
+-- If it is, read it and insert its contents in its place
+expandFileThings :: [String] -> IO [String]
+expandFileThings = concatMapM isThingFile
+  where
+    isThingFile :: String ->  IO [String]
+    isThingFile thing = do
+        isFile <- doesFileExist thing
+        if isFile then
+            lines <$> readFile thing
+        else
+            return [thing]
```

Comment:
I think this could be reworked as:

```haskell
import Control.Conditional(ifM)

isThingFile thing = ifM (doesFileExist thing)
                        (lines <$> readFile thing)
                        (return [thing])
```

You might need to screw with the types somewhat, especially on the true branch.